mycore-base/src/main/java/org/mycore/common/config/MCRConfigurationException.java

```diff
@@ -28,6 +28,7 @@
  * @author Frank Lützenkirchen
  */
 public class MCRConfigurationException extends MCRException {
+
```

Comment:
unnecessary

Reply:
maybe not necessary, but fixing a formatting issue as per [Boy Scout Rule](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)